common/source/docs/common-ac2_simple_geofence.rst

```diff
@@ -1,22 +1,22 @@
 .. _common-ac2_simple_geofence:
 
-[copywiki destination="copter,rover"]
+
 =================
 Cylindrical Fence
 =================
 
 Overview
 ========
 
-Copter and Rover include a simple "tin can" shaped fence centered
+ArduPilot includes a simple "tin can" shaped fence centered
 on home that will attempt to stop your copter/rover from flying/driving too far away
-by stopping at the fence or executing a configurable action.
-The maximum circular distance and altitude (altitude used by Copter only) and the vehicle behavior when the fence is reached can be configured using Mission Planner.
+by stopping at the fence some Copter/Rover modes or executing a configurable action.
+The maximum circular distance and altitude (altitude used by Plane and Copter only) and the vehicle behavior when the fence is reached can be configured using Mission Planner.
```

Comment:
It can be configured using any GCS - should we be stating only mission planner here?

Reply:
done